src/aap_eda/api/serializers/decision_environment.py

```diff
@@ -70,6 +82,26 @@ class DecisionEnvironmentCreateSerializer(serializers.ModelSerializer):
             validators.check_credential_registry_username_password,
         ],
     )
+    pull_policy = serializers.CharField(
+        required=False,
+        allow_blank=False,
+        default=settings.DEFAULT_PULL_POLICY,
+    )
+
+    def validate_pull_policy(self, value):
+        """Normalize pull policy input and validate."""
+        if value:
+            normalized = convert_pull_policy_from_frontend(value)
+            if normalized not in ImagePullPolicy.values():
+                valid_options = ["always", "never", "missing"] + list(
+                    ImagePullPolicy.values()
+                )
+                raise serializers.ValidationError(
```

Comment:
Why show to the user 6 total options? We should show only 3 options. The backend can convert them in a case insensitive way.